packages/atomic/cypress/integration/result-list/result-list.cypress.ts

```diff
@@ -1,158 +1,208 @@
-    beforeEach(() => {
-      new TestFixture()
-        .withoutFirstAutomaticSearch()
-        .with(addResultList())
-        .init();
-    });
-
-    it('should render placeholder components', () => {
-      ResultListSelectors.placeholder().should('be.visible');
-    });
-  });
-
-  describe('when an initial search is executed', () => {
-    it('should render the correct number of results', () => {
-      new TestFixture().with(addResultList()).init();
-      ResultListSelectors.result().should('have.length', 10);
-    });
-  });
-
-  describe('when multiple searches are executed', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .with(addResultList())
-        .withElement(generateComponentHTML(pagerComponent))
-        .init();
-    });
-
-    it('should update the results', () => {
-      let firstResultHtml: string;
-
-      ResultListSelectors.firstResult().then((element) => {
-        firstResultHtml = element[0].innerHTML;
-      });
-
-      PagerSelectors.buttonNext().click();
-      cy.wait(500);
-
-      ResultListSelectors.firstResult().should((element) => {
-        const secondResultHtml = element[0].innerHTML;
-        expect(secondResultHtml).not.to.equal(firstResultHtml);
-      });
-    });
-  });
-
-  describe('with elements to measure line height', () => {
-    const lineHeightSelector = '#line-height-el';
-
-    function generateLineHeightElement() {
-      return generateComponentHTML('div', {
-        id: lineHeightSelector.slice(1),
-        style:
-          'background-color: red; width: var(--line-height, 0); height: var(--line-height, 0);',
-      });
-    }
-
-    before(() => {
-      new TestFixture()
-        .with(
-          addResultList(
-            buildTemplateWithSections({
-              title: generateLineHeightElement(),
-              excerpt: generateLineHeightElement(),
-              bottomMetadata: generateLineHeightElement(),
-            })
-          )
-        )
-        .init();
-      ResultListSelectors.shadow().find('.list-wrapper:not(.placeholder)');
-    });
-
-    withAnySectionnableResultList((display, imageSize, density) => {
-      it('should expose --line-height in the title section', () => {
-        ResultListSelectors.sections
-          .title()
-          .find(lineHeightSelector)
-          .should('be.visible');
+const foldedResultListConfig = [
+  FoldedResultListSelectors,
+  foldedResultListComponent,
+  addFoldedResultList,
+  'Folded result List Component',
+  true,
+];
+const resultListConfig = [
+  ResultListSelectors,
+  resultListComponent,
+  addResultList,
+  'Result List Component',
+  false,
+];
+
+const configs: Array<
+  [
+    typeof ResultListSelectors,
+    string,
+    (template?: HTMLElement | undefined) => (fixture: TestFixture) => void,
+    string,
+    boolean
+  ]
+> = [foldedResultListConfig, resultListConfig];
+
+configs.forEach(
+  ([componentSelectors, componentTag, addResultFn, title, isFolded]) => {
```

Comment:
I think `isFolded` could be omitted from the config, and just derived from `componentTag === 'atomic-folded-result-list'`

Reply:
coolz, will change